test/mock/kernel/vga_mock.zig

```diff
@@ -0,0 +1,102 @@
+const std = @import("std");
+const expect = std.testing.expect;
+
+const arch = @import("arch.zig").internals;
+const mock_framework = @import("mock_framework.zig");
+
+pub const WIDTH: u16        = 80;
+pub const HEIGHT: u16       = 25;
+
+pub const COLOUR_BLACK: u4            = 0x00;
```

Comment:
Hmm, can we avoid duplicating all of these across this file and the normal vga file?

Reply:
I'll see what I can do

Reply:
I don't think I can. I did this bc you can redefine functions so can't import the original vga.zig file and just redefine the functions and leave the consts.

Reply:
That makes sense. We'll just have to remember to change them here when changing them in vga.zig